larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -172,11 +246,13 @@ inline pandora::StatusCode LArMCParticleFactory::Write(const Object *const pObje
     {
         pandora::BinaryFileWriter &binaryFileWriter(dynamic_cast<pandora::BinaryFileWriter&>(fileWriter));
         PANDORA_RETURN_RESULT_IF(pandora::STATUS_CODE_SUCCESS, !=, binaryFileWriter.WriteVariable(pLArMCParticle->GetNuanceCode()));
+        PANDORA_RETURN_RESULT_IF(pandora::STATUS_CODE_SUCCESS, !=, binaryFileWriter.WriteVariable(static_cast<int>(pLArMCParticle->GetProcess())));
```

Comment:
Should probably add version checking to the output too, if we're rewriting from an old pndr file for some reason, the process still won't exist

Reply:
And a brief request to @johnmarshall80 to take a look at the LArMCParticle.h changes in particular, given I had input into the changes to this file, having an additional set of eyes for the review would be useful. Thanks!